backend/src/main/java/rocks/inspectit/gepard/agentmanager/connection/model/dto/ConnectionDto.java

```diff
@@ -0,0 +1,33 @@
+/* (C) 2024 */
+package rocks.inspectit.gepard.agentmanager.connection.model.dto;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import lombok.Builder;
+import rocks.inspectit.gepard.agentmanager.connection.model.Connection;
+
+/** Represents a connection response. */
+@Builder
+public record ConnectionDto(
+    UUID id,
+    LocalDateTime registrationTime,
+    String serviceName,
+    String gepardVersion,
+    String otelVersion,
+    Long pid,
+    Long startTime,
+    String javaVersion) {
```

Comment:
Shouldn't these properties also be annotated with `@NotNull`?

Reply:
Done.